### ISSUE TITLE
为什么验证过程中预测结果和标签需要转移到cpu上？

### ISSUE THREAD

**Issue by Xingmao-w** [thuml/Time-Series-Library#695]
`batch_x = batch_x.float().to(self.device)`
`batch_y = batch_y.float()`
在验证过程中，batch_y 就没有转移到GPU上，这有什么深意吗，还是忘记了？

`pred = outputs.detach().cpu()`
`true = batch_y.detach().cpu()`
后面验证过程中，预测结果和标签又转移到了CPU上，这是为什么

**Comment by wuhaixu2016** [comment]
您好，没有什么特殊设计，您也可以使用gpu计算
